### PR TITLE
fixed pages paths

### PR DESCRIPTION
**Pull request by krishnaglodha**
## Launch Checklist

 - [ ] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [ ] Briefly describe the changes in this PR.
 - [ ] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [ ] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

File: docs/src/components/style-spec/specs/item.jsx [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+                    <div className="mt12">
+                        <SDKSupportTable {...this.props['sdk-support']} />
+                    </div>
+                )}
+            </React.Fragment>
+        );
+    }
+}
+
+Item.propTypes = {
```
Comment (birkskyum) [inline docs/src/components/style-spec/specs/item.jsx:341]: Can we use typescript instead of PropTypes?

File: docs/package.json [hunk 2]
```diff
@@ -10,20 +10,28 @@
...
-		"react-dom": "^18.1.0",
-		"@astrojs/react": "^2.0.2",
-		"@astrojs/preact": "^2.0.3",
 		"@algolia/client-search": "^4.13.1",
+		"@astrojs/mdx": "^0.17.2",
+		"@astrojs/preact": "^2.0.3",
+		"@astrojs/react": "^2.0.2",
 		"@docsearch/css": "^3.1.0",
 		"@docsearch/react": "^3.1.0",
-		"@types/react": "^17.0.45",
```
Comment (birkskyum) [inline docs/package.json:22]: These @types/* can go to  devDependencies

File: docs/src/components/style-spec/specs/item.jsx (hunk 1 above)
Comment (birkskyum) [reply docs/src/components/style-spec/specs/item.jsx:341]: Runtime type checking is not so meaningful for SSG sites where everything happens at build-time, whereas improved static-analysis can be very helpful.

File: docs/package.json (hunk 2 above)
Comment (krishnaglodha) [reply docs/package.json:22]: moved

File: docs/src/components/style-spec/specs/item.jsx (hunk 1 above)
Comment (birkskyum) [reply docs/src/components/style-spec/specs/item.jsx:341]: yes

Comment (krishnaglodha) [reply docs/src/components/style-spec/specs/item.jsx:341]: ohh okay, I'll have to make all files tsx then

Comment (krishnaglodha) [reply docs/src/components/style-spec/specs/item.jsx:341]: actually I don;t have much experience with typescript tbh

Comment (birkskyum) [reply docs/src/components/style-spec/specs/item.jsx:341]: It's a superset of js so changing the files tsx should be correct syntax still

Comment (krishnaglodha) [reply docs/src/components/style-spec/specs/item.jsx:341]: Hi yeah I tried it, but it is giving me following type of errors
Screenshot 2023-03-07 at 11 33 29 AM

I'm guessing these are the errors related to typescript data type definitions right?

Comment (birkskyum) [reply docs/src/components/style-spec/specs/item.jsx:341]: Yes props need an interface like this

https://www.benmvp.com/blog/react-prop-types-with-typescript/

**Pull request closed without merging**